code-challenges.test.js

```diff
@@ -15,6 +19,29 @@
 
 // a) Create a test with an expect statement using the variable provided. HINT: Check out this resource: https://jestjs.io/docs/expect#expectarraycontainingarray
 
+//Pseudocode
+// -First use the 'describe' method name the function 'colors'
+// -Then use the 'it' method to describe the function: takes in an array, removes the first item from the array and shuffles the remaining content.
+// - Then list the the variables to perform the test on 'const colors1 and colors2'
+// - Then use the 'expect' method to call the function in the describe method 'colors' for each variable 'colors1 and colors2'
+// - Then use the '.toEqual' method to list the expected output of test. Removed first item in each array
+// - Create the function 'colors'
+// - Create parameter (array) to pass any array
+// - Use HOF .map to iterate through each array
+// - Use 'shuffle method to return random order
+
+describe("colors", () => {
+    it("takes in an array, removes the first item from the array and shuffles the remaining content", () => {
+        const colors1 = ["purple", "blue", "green", "yellow", "pink"]
+        const colors2 = ["chartreuse", "indigo", "periwinkle", "ochre", "aquamarine", "saffron"]
+        expect(colors(colors1)).toEqual(shuffle["blue", "green", "yellow", "pink"])
+        expect(colors(colors2)).toEqual(shuffle["indigo", "periwinkle", "ochre", "aquamarine", "saffron"])
+
+    })
+    
+}
+)
```

Comment:
Your test structure looks great!  This problem is really tricky because the output will be random and not consistently the same output.  There is something you can add to the .toEqual match that allows you to check if the array contains the expected output, instead of strictly equal.